foyer/version.py

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
We don't want to merge this file (in fact, version.py should probably be added to .gitignore).